.travis.yml

```diff
@@ -3,9 +3,16 @@ android:
   components:
     - platform-tools
     - tools
-
     - build-tools-21.1.0
-
     - android-21
-
     - extra-google-google_play_services
+
+deploy:
+  provider: releases
+  api_key:
+    secure: Si6dy+ETFBwsbbjMOd9WI8ho0TMDnL46/vhKxLc5Rlvh75UUZH/ThEht/eRhZBP4IgEyg1oeR/8NOL8Lg70aA/cnv0c1+3u050MrUgDIMbbVwgo/l1v7CXfHvRYmicwcn5l9+qhcVaS3Lp7rftfeEvUgXLwwPS0kRxHZuINIYw8=
+  file: App/build/outputs/apk/App-debug.apk
+  skip_cleanup: true
+  on:
+    tags: true
+    all_branches: true
```

Comment:
Do we want have `all_branches: true`? I don't think it matters on TheSoftwareFactory/lokki-android since there shouldn't be other branches anyway if all work is done through pull requests, but I wonder what implications it has for forks – this generates releases on forks too to their own repository if they happen to have Travis CI enabled, right? At least it would be cool if release name would include branch it's generated from to avoid any confusion – even though it's kinda implictly inside the commit SHA, which seem to be included.

Anyway GJ :-)